pandas/io/excel/_base.py

```diff
@@ -302,6 +304,11 @@
 read_csv : Read a comma-separated values (csv) file into DataFrame.
 read_fwf : Read a table of fixed-width formatted lines into DataFrame.
 
+Notes
+-----
+For specific information on the methods used for each Excel engine, refer to the pandas
+user guide: https://pandas.pydata.org/docs/user_guide/io.html#excel-files.
```

Comment:
This is an internal cross reference, don't use an absolute URL.

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html